src/encoding.ts

```diff
@@ -1,4 +1,4 @@
-import { jschardet } from "./vscodeModules";
```

Comment:
Could you try use webpack without change this?

Reply:
Yesh will try

Reply:
getting
```
WARNING in ./src/vscodeModules.ts 18:15-26
Critical dependency: the request of a dependency is an expression
 @ ./src/svn.ts
 @ ./src/extension.ts
```
I dont know enough about webpack to fix it

Reply:
I will try later